antarest/study/storage/rawstudy/model/filesystem/root/output/simulation/mode/mcall/digest.py

```diff
@@ -0,0 +1,131 @@
+# Copyright (c) 2024, RTE (https://www.rte-france.com)
+#
+# See AUTHORS.txt
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of the Antares project.
+
+import typing as t
+
+import pandas as pd
+
+from antarest.core.model import JSON
+from antarest.core.serialization import AntaresBaseModel
+from antarest.study.storage.rawstudy.model.filesystem.config.model import FileStudyTreeConfig
+from antarest.study.storage.rawstudy.model.filesystem.context import ContextServer
+from antarest.study.storage.rawstudy.model.filesystem.root.output.simulation.mode.mcall.synthesis import OutputSynthesis
+
+
+class DigestMatrixDTO(AntaresBaseModel):
+    columns: t.List[t.Union[str, t.List[str]]]
+    data: t.List[t.List[t.Any]]
+    groupedColumns: bool
+
+
+class DigestDTO(AntaresBaseModel):
+    area: DigestMatrixDTO
+    districts: DigestMatrixDTO
+    flowLinear: DigestMatrixDTO
```

Comment:
For the flows data, I am not really not sure a matrix will be the best representation (in the data, not in the front), because by nature it will be very sparse.

Basically we have 2 values per-link, it seems to me it would make more sense to have a structure such as:
```json
[
    {
        "from": "fr",
        "to": "be",
        "direct_flow": 100,
        "opposite_flow": -100,
        "direct_quad_flow": 20,
        "opposite_quad_flow": -20,
    },
    ...
]
```